tests/Http/TagsTest.php

```diff
@@ -17,6 +16,9 @@ class TagsTest extends TestCase
      */
     public function testTaggingAndUntaggingAPost()
     {
+        // Supress sending Slack notifications for "good submission" tagging.
+        $this->withoutNotifications();
```

Comment:
I left this test tagging post with `Good Submission` and using the `withoutNotifications()` method as an example so we know it's an option to suppress notifications when testing and since this test mentioned creating a new event anyhoo.

Reply:
Let's disable this in tests across the board by setting `SLACK_WEBHOOK_INTEGRATION_URL` to an empty string in our `phpunit.xml` config! That way we don't need to suppress test-by-test.

Reply:
Ah nice, ok. 

Only caveat is if we did want to write a specific test for Notifications we wouldn't be able to since it'd be dependent on this env var being present 🤔